src/executor/disk_based_storage_executor.py

```diff
@@ -32,17 +32,22 @@ class DiskStorageExecutor(AbstractStorageExecutor):
 
     def __init__(self, node: StoragePlan):
         super().__init__(node)
+        """
         self.storage = Loader(node.video,
                               batch_size=node.batch_size,
                               skip_frames=node.skip_frames,
                               limit=node.limit,
                               offset=node.offset,
                               curr_shard=node.curr_shard,
                               total_shards=node.total_shards)
+        """
```

Comment:
We should get away with these comments. And do you think we still need a disk-based storage executor? Our storage engine should ideally replace this.

Reply:
Yeah, we can get rid of it.